src/char.cr

```diff
@@ -873,10 +873,14 @@ struct Char
   # 'あ'.to_s # => "あ"
   # ```
   def to_s : String
-    String.new(4) do |buffer|
-      appender = buffer.appender
-      each_byte { |byte| appender << byte }
-      {appender.size, 1}
+    bytesize = self.bytesize
+    String.new(bytesize) do |pointer|
+      i = 0
+      self.each_byte do |byte|        
```

Comment:
No need for `self` (and trailing whitespace 😅 )

```suggestion
      each_byte do |byte|
```

Reply:
I honestly have no idea where that whitespace came from, thanks!